app/src/main/java/com/monkeyteam/chimpagne/model/feed/FeedAlgorithm.kt

```diff
@@ -0,0 +1,21 @@
+package com.monkeyteam.chimpagne.model.feed
+
+import com.monkeyteam.chimpagne.model.database.ChimpagneEvent
+import com.monkeyteam.chimpagne.model.location.Location
+
+val N_CLOSEST = 4
```

Comment:
This value is ridiculously low, why do we need to take only the N_CLOSEST events ?

Reply:
I mean 1000000 events would be too much for example.
We have to take a finite amount. Changing the algorithm is not part of this pr.

Reply:
Why would it be too much ? From a performance perspective, you're already sorting those 1000000 events locally. LazyColumn is made to display a huge amount of items.